plugins/govppmux/govpp_channel.go

```diff
@@ -0,0 +1,122 @@
+package govppmux
+
+import (
+	"time"
+
+	govppapi "git.fd.io/govpp.git/api"
+	"git.fd.io/govpp.git/core"
+	"github.com/ligato/cn-infra/logging/logrus"
+)
+
+const defaultRetryRequestTimeout = 500 * time.Millisecond
+
+// goVppChan implements govpp channel interface. Instance is returned by NewAPIChannel() or NewAPIChannelBuffered(),
+// and contains *govpp.channel dynamic type (vppChan field). Implemented methods allow custom handling of low-level
+// govpp.
+type goVppChan struct {
+	vppChan govppapi.Channel
```

Comment:
You could just embed `govppapi.Channel` interface so you don't have to re-implement each of its methods.